instill-ai/vdp
# chore(model): add real models test

Because

- integration tests do not cover the real production models

This commit

- add test cases for testing the real production models

How to test:
There are 3 environments that should be tested with this PR:

Macbook M1:
```bash
clone VDP repository https://github.com/instill-ai/vdp.git

launch VDP: make build; make dev PROFILE=all

run the script test: cd test; make test-no-large-models MODE=localhost TEST_CPU_ONLY=true TARGET=m1 
```

- Local environment with Nvidia GPU

```
clone VDP repository https://github.com/instill-ai/vdp.git 

launch VDP: make build; make dev PROFILE=all

run the script test: cd test; make test-all MODE=localhost
```

- CI/CD:

There are 2 GitHub Actions running:
- Test the models with `MODE=demo` and `TEST_GPU_ONLY=true`
- Test the models on the Ubuntu environment with `MODE=localhost` and `TEST_CPU_ONLY=true`

The result is show on GitHub Actions (will update with inform result via slack)

fix INS-179
